README.md

```diff
@@ -133,7 +133,7 @@ Crossplane is under the Apache 2.0 license.
 <!-- Named links -->
 
 [Crossplane]: https://crossplane.io
-[Install Crossplane]: docs/getting-started/install-configure.md
+[Install Crossplane]: https://crossplane.io/docs/latest
```

Comment:
Both Getting Started and Docs link the same page one after the other now.

Reply:
@muvaf right, it's more opportunities to get into the docs and to the website

Reply:
There are `Getting Started` and `Docs` links under `Getting Started & Documentation` section and they both take you to the same page.